idunn/api/utils.py

```diff
@@ -18,6 +20,14 @@
     ]
 }
 
+ANY = "_any"
+
+def send_400(msg):
+    raise BadRequest(
```

Comment:
it would be more explicit to define that as a new exception class, derived from `BadRequest`.